.travis.yml

```diff
@@ -1,6 +1,6 @@
 sudo: required
 language: node_js
-node_js: '8'
+node_js: '10'
```

Comment:
So our base line now nodejs 10? If not it should stay nodejs 8, we should avoid using any nodejs 10 apis

Reply:
It's just a test to verify that the other commit fixes Theia for Node 10. I'll revert it once the tests pass.